code/game/machinery/medical_kiosk.dm

```diff
@@ -237,7 +237,7 @@
 			if(R.overdosed)
 				overdose_list += list(list("name" = R.name))
 	var/obj/item/organ/stomach/belly = altPatient.getorganslot(ORGAN_SLOT_STOMACH)
-	if(belly?.reagents.reagent_list.len) //include the stomach contents
+	if(belly && belly.reagents.reagent_list.len) //include the stomach contents if it exists
```

Comment:
```suggestion
	if(belly?.reagents.reagent_list.len) //include the stomach contents if it exists
```
This is fine. I was just asking if both the `reagent` datum and `reagent_list` always exist.
If so, keep it like this.